app/src/main/java/com/kickstarter/ui/SharedPreferenceKey.java

```diff
@@ -0,0 +1,13 @@
+package com.kickstarter.ui;
+
+public final class SharedPreferenceKey {
```

Comment:
technically we don't need this class of static key values since these keys are only used once...but it is nice to consolidate them in one place so we know what values we're storing in SharedPreferences at a glance!